### PR TITLE
Write cli I/O output data to file

### PR DESCRIPTION
**Pull request by nmlinaric**
## Description
Added function which writes cli I/O data to log file.

## Related Issue Or Context

Closes: #74 

## How Has This Been Tested? Testing details.
Wrote unit test for writing cli I/O data to file

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation

## Checklist:
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have ensured that all acceptance criteria (or expected behavior) from issue are met
- [ ] I have updated the documentation locally and in chainbridge-docs.
- [x] I have added tests to cover my changes.
- [x] I have ensured that all the checks are passing and green, I've signed the CLA bot

### CODE REVIEW

File: chains/evm/cli/cli.go [hunk 1]
```diff
@@ -52,6 +61,62 @@ func BindEVMCLIFlags(evmRootCLI *cobra.Command) {
 
 }
 
+func writeCliIODataToFile(cmd *cobra.Command) {
```
Comment (P1sar) [inline chains/evm/cli/cli.go:64]: I do not see that this function been used in all CLIs, means now its just sits here and after merge no logging will be happen, or i am missing something?

File: chains/evm/cli/cli.go [hunk 2]
```diff
@@ -52,6 +61,62 @@ func BindEVMCLIFlags(evmRootCLI *cobra.Command) {
...
+	file, err := os.OpenFile(cliLogsFilename, os.O_APPEND|os.O_CREATE|os.O_WRONLY, 0644)
+	if err != nil {
+		log.Error().Err(fmt.Errorf("failed to create cli log file: %v", err))
+	}
+
+	defer file.Close()
+
+	currentTimestamp := time.Now().Format("02-01|15:00.000 ")
+
+	urlFlagName := cmd.Flag(UrlFlagName).Name
```
Comment (P1sar) [inline chains/evm/cli/cli.go:74]: i seemse that you are parsing only global flags, but what will happen when i will call for example:
```./chainbridge-core-example \
evm-cli \
bridge \
register-resource \
--url $NODE_ENDPOINT_NETWORKA \
--privateKey BRIDGE_ADMIN_PRIVATE_KEY \
--bridge 0x0 \
--handler 0x0 \
--target 0x0 \
--resourceId 000000000000000000000000000000e389d61c11e5fe32ec1735b3cd38c69500```
I will not see the exact name of CLI that was called (register-resource) as well as there will not be any CMD specific args

File: chains/evm/cli/writer/writer_test.go [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+
+func (s *WriterTestSuite) SetupSuite()    {}
+func (s *WriterTestSuite) TearDownSuite() {}
+
+func (s *WriterTestSuite) TearDownTest() {}
+
+func (s *WriterTestSuite) TestWriteCliDataToFile() {
+	expectedLog := time.Now().Format("02-01|15:00.000 ") +
+		"Passed flags: gasLimit, gasPrice, help, jsonWallet, jsonWalletPassword, networkid, url,  with args: 7000000, 25000000000, false, test-wallet, test-wallet-password, 0, test-url, =>\n" +
+		"Called evm-cli with args: --gasLimit=\"7000000\" --gasPrice=\"25000000000\" --help=\"false\" --jsonWallet=\"test-wallet\" --jsonWalletPassword=\"test-wallet-password\" --networkid=\"0\" --url=\"test-url\" \n"
```
Comment (P1sar) [inline chains/evm/cli/writer/writer_test.go:31]: Format: {09-01|14:30:21.165} {cmd and args used} => (new line) {finalized log}
example of finalaized log https://github.com/ChainSafe/chainbridge-core/blob/main/chains/evm/cli/erc20/deposit.go#L123

Idea is that WriteCliDataToFile should also accept some string that holds finalised logs, main idea of such a behaviour is to have stored resulted logs that could contain useful information, like addresses of deployed smart contracts or transaction hashes for debug

**Review by P1sar — CHANGES_REQUESTED** [review]

File: chains/evm/cli/account/generate-key-pair.go [hunk 4]
```diff
@@ -11,6 +12,10 @@ var generateKeyPairCmd = &cobra.Command{
 	Short: "Generate bridge keystore (Secp256k1)",
 	Long:  "The generate subcommand is used to generate the bridge keystore. If no options are specified, a Secp256k1 key will be made.",
 	RunE:  generateKeyPair,
+	PreRun: func(cmd *cobra.Command, args []string) {
+		confirmTransfer(cmd, args)
```
Comment (P1sar) [inline chains/evm/cli/account/generate-key-pair.go:16]: Why we need to confirm this command explicitly?

**Review by P1sar — CHANGES_REQUESTED** [review]

File: chains/evm/cli/account/generate-key-pair.go (hunk 4 above)
Comment (P1sar) [reply chains/evm/cli/account/generate-key-pair.go:16]: Everything else looks great

Comment (nmlinaric) [reply chains/evm/cli/account/generate-key-pair.go:16]: This shouldn't be here, seems like copy-paste mistake :sweat:

**Review by P1sar — APPROVED** [review]

**Pull request merged**